os/board/common/partitions.c

```diff
@@ -34,6 +34,15 @@
 #include <tinyara/fs/mksmartfs.h>
 #endif
 
+#if defined(CONFIG_LIB_BOARDCTL) && defined(CONFIG_BOARDCTL_RESET)
```

Comment:
`CONFIG_BOARDCTL_RESET` is enabled only when `CONFIG_LIB_BOARDCTL` is enabled.
So I think we can check only with `CONFIG_BOARDCTL_RESET`.

Reply:
thanks.